sklearn/linear_model/logistic.py

```diff
@@ -1310,7 +1304,8 @@ class LogisticRegression(BaseEstimator, LinearClassifierMixin,
     max_iter : int, optional (default=100)
         Maximum number of iterations taken for the solvers to converge.
 
-    multi_class : str, {'ovr', 'multinomial', 'auto'}, optional (default='ovr')
+    multi_class : str, {'ovr', 'multinomial', 'auto'}, optional \
```

Comment:
remove the "str, " and fit this all on one line? (Personally, I think `optional` and `default=...` is also redundant obsolescence.)